sessions/js-functions/pizza-analyzer/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "env": {
+    "browser": true,
+    "es2022": true,
+    "node": true
+  },
+  "extends": ["eslint:recommended"],
```

Comment:
should this include `"prettier"`?

Reply:
I am not sure, I used the template from the new format team. I guess it should be discussed with them.

Reply:
not nessesary unless you want to use eslint as a formatter :)